PerfectXL.EPPlus/Style/ExcelColor.cs

```diff
@@ -268,9 +268,13 @@ public static string LookupColor(ExcelColor theColor, IList<SchemeColor> schemeC
                 var hexValue = schemeColors?.ElementAtOrDefault(index)?.Value ?? "818181"; // arbitrary sensible value
                 rawColorString = $"#FF{hexValue}";
             }
+            else if (theColor.Indexed == null)
+            {
+                rawColorString = "#FF808080";

```

Comment:
Shouldn't `LookupColor` also return null? Default Lookupcolor for backgroundcolor should be white while default LookupColor for cell font should be black.

Reply:
Yes!